scripts/docker/dockerfile_clang-10

```diff
@@ -0,0 +1,30 @@
+FROM axom/compilers:clang-10
+
+# Update base image so we don't have to install so many things via Spack
+SHELL ["/bin/bash", "-c"]
+RUN sudo apt-get update -y
+RUN sudo apt-get install curl ssh pkg-config build-essential libopenblas-dev mpich python3 cmake -fy
+RUN sudo apt-get -qq install -y --no-install-recommends gfortran-7 && sudo update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-7 100
+RUN sudo ln -f -s python3 /usr/bin/python # Compatibility with legacy Python
+
+WORKDIR "/home/axom"
+USER axom
+
+# Clone axom
+RUN git clone -b feature/white238/addclang10docker --recursive https://github.com/LLNL/axom.git
```

Comment:
> RUN git clone -b feature/white238/addclang10docker --recursive https://github.com/LLNL/axom.git 

This line doesn't seem right for the checked-in version.

Reply:
you have to put the current branch you are working with otherwise you cant make changes without pushing to develop.  Once we are on github actions you can pass them via argument.

Reply:
I think i could do it this way:

`docker build --build-arg GIT_BRANCH=feature/white238/addclang10docker -t "axom/tpls:clang-10" - < dockerfile_clag-10` 

Then in the dockerfile:

`ARG branch=develop` // this defaults to develop I'm told